jekyll/_cci2/images/linux-vm/14.04-to-20.04-migration.md

```diff
@@ -16,6 +16,50 @@ version:
 * TOC
 {:toc}
 
+## Switching to a supported image
+{: #switching-to-a-supported-image }
+
+Swapping out the deprecated image for a supported one in the CircleCI config is straight forward.
+Additional work to get your pipelines to run with the newer image will depend on if the [changes below](#changes) apply to your project.
```

Comment:
Are we able to provide any further details or maybe resources about what this additional work would entail?

Reply:
Nope. That's the hard part. It all depends on what they're doing and how they use the 25+ different bits of software that are pre-installed in the image.

Overtime, the plan is to see the most common items customers run into converting, and then add them to these guides as we go so that they become more useful.

Reply:
Added a note to both docs. Let me know what you think.